.github/workflows/build-and-deploy-test.yml

```diff
@@ -32,7 +31,9 @@ jobs:
       TEST_NGINX_BASIC_AUTH: ${{ secrets.TEST_NGINX_BASIC_AUTH }}
       PIPELINE_URL: https://github.com/${{ github.repository }}/actions/runs/${{ github.run_id }}
       SENTRY_DSN: ${{ secrets.SENTRY_DSN_FRONTEND }}
-      PIPELINE_URL: https://github.com/${{ github.repository }}/actions/runs/${{ github.run_id }}
```

Comment:
This is used in info target [here](https://github.com/IntersectMBO/govtool/blob/develop/scripts/govtool/info.mk#L2).
Without that you'll have no link to the pipeline in slack notification.